verifiers/utils/builder.go

```diff
@@ -132,7 +132,11 @@ func IsValidBuilderTag(ref string, testing bool) error {
 	if testing {
 		// Tags on trusted repositories should be a valid semver with version
 		// core including all three parts and no build identifier.
-		versionCore := strings.Split(pin, "-")[0]
+		parts := strings.Split(pin, "-")
+		if len(parts) == 0 {
```

Comment:
nit: I'm pretty sure `strings.Split` will never return a slice with a length of zero (if `sep` is non-empty, which it is in this case). Was this an issue you actually encountered?

Reply:
just read the doc and you're right. Will revert.